ResumableUpload.VB.Sample/ResumeableUploadSample.vb

```diff
@@ -0,0 +1,205 @@
+﻿Imports System
+Imports System.IO
+Imports System.Reflection
+Imports Google.Apis.Http
+Imports Google.Apis.Requests
+Imports Google.Apis.Services
+Imports Google.Apis.Util
+Imports Google.Apis.YouTube.v3
+Imports Google.Apis.YouTube.v3.Data
+Imports Google.Apis.Auth.OAuth2
+Imports Google.Apis.Upload
+Imports System.Net.Http
+Imports System.Threading
+Imports System.Threading.Tasks
+
+Module ResumeableUploadSample
+    ''' <summary>
+    ''' A VB .NET sample for the YouTube Data API using ResumableUpload.
```

Comment:
My feedback for this code is pretty much the same as the C# version.

- Don't use hungarian notation.
- Splitting up the `ExecuteUploadProcess` into smaller pieces would help readability.